docs/lang/articles/basic/overview.md

```diff
@@ -2,22 +2,48 @@
+
+The concept of decoupling is further extended to the type system. With GPU memory capacity and bandwidth becoming the major bottlenecks nowadays, it is vital to be able to pack more data per memory unit. Since 2021, Taichi has introduced customizable quantized types, allowing for the definition of fixed point or floating point numbers with arbitrary bits (still needs to be under 64). This has allowed an MPM simulation of over 400 million particles on a single GPU device. Learn more details in [the QuanTaichi paper](https://yuanming.taichi.graphics/publication/2021-quantaichi/quantaichi.pdf).
+
+Taichi is intuitive. If you know Python, you know Taichi. If you write Taichi, you awaken your GPU (or CPU as a fallback). Ever since its debut, this simple idea has gained so much popularity, that many were attracted to contribute new backends, including Vulkan, OpenGL and DirectX (working in progress). Without our strong and dedicated community, Taichi would never have been where it is now.
+
+Going forward, we see many new opportunities lying ahead this path, and would like to share some of our vision with you.
+
+**Academia**
+
+Taichi will keep embracing the academia. Towards this goal, we have implemented or planned for features that are particular useful for researchers, small matrix and sparse matrix solvers, to name a few. Taichi has also supported automatic differentiation, a key feature for differential simulation and robotics training.
+
+
+**Apps & game engine integration**
+
+One huge advantange of Taichi lies in its portability, thanks to the support for a wide variety of backends. During the development process, we have also recognized the increasing demands from our industry users for multi-platform packaging and deployment. Below shows an experimental demo of integrating Taichi with Unity. By exporting Taichi kernels as SPIR-V shaders, we can easily import them into a Unity project.
+
+![](https://github.com/taichi-dev/taichi_assets/blob/master/static/imgs/unity_fluid.gif?raw=true)
+
+**General-purpose computing**
+
+While originally designed for physics simulation, Taichi has found its application in many other areas that can be boosted by GPU  general-purpose computing.
+
+* [taichimd](https://github.com/victoriacity/taichimd): Interactive, GPU-accelerated Molecular (& Macroscopic) Dynamics using the Taichi programming language
+* [TaichiSLAM](https://github.com/xuhao1/TaichiSLAM): a 3D Dense mapping backend library of SLAM based Taichi-Lang, designed for the aerial swarm.
+* [Stannum](https://github.com/ifsheldon/stannum): Fusing Taichi into PyTorch.
+
```

Comment:
Maybe we can also mention the fluid solver here? https://github.com/hietwll/LBM_Taichi

Reply:
I was hoping to use some cases that are not simulation LOL